### PR TITLE
Add details 

### PR DESCRIPTION
**Pull request by tamai2505**
This add some much needed details to the README

### CODE REVIEW

**Comment by dsorkin** [comment]
:+1: Looks good.

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+## section 1
+
+ Brief summary comment of the project goals.
+ 
+## section 2
+
+*list item one
+*list item two
+
+![](https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcTsAxzq51FyPJX7pxMAqYEPD9R52__EJpoDsqFe6fiZr9viyot22SHhGA_w)
```
Comment (dsorkin) [inline README.md:12]: Good markdown syntax

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
 # hello-world
+
+## section 1
+
+ Brief summary comment of the project goals.
```
Comment (tamai2505) [inline README.md]: thank you consideration

**Comment by tamai2505** [comment]
merge #1

**Pull request merged**